core/debug/src/debug.c

```diff
@@ -24,12 +24,12 @@
 #include <stdbool.h>
 
 #define DEBUG_PRINT_HEAD(x) {\
-    DPRINTF("\n***********************************************************\n");\
-    DPRINTF("                    "x"\n");\
-    DPRINTF("***********************************************************\n\n");\
+    DPRINTF("\r\n***********************************************************\r\n");\
```

Comment:
please remove all \r and configure your serial terminal to force CR on each LF - it is a terminal configuration and should not be treated in code.

Reply:
Done.